src/index.html

```diff
@@ -5,10 +5,70 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
+
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="moyo-logo-link">
+        <img class="moyo-logo"
+        src="/images/logo.png"
+        alt="moyo logotype"
+        >
+      </a>
+
+      <nav class="header-nav">
+        <ul class="nav-list">
+          <li class="nav-list-item">
```

Comment:
`nav-list-item` can not be block
because a list element without a list does not exist
`nav-list-item` is an element of the block `nav-list`
so [Follow naming convention `block__element--modifier`](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#bem)